src/test/java/bio/overture/ego/grpc/service/UserServiceGrpcAuthTest.java

```diff
@@ -104,6 +105,8 @@ public void testDataSetup() {
                   .lastName("")
                   .firstName("")
                   .type(UserType.USER)
+                  .identityProvider(GOOGLE)
```

Comment:
the Ego.proto file needs to be updated with these changes!
https://github.com/overture-stack/ego/blob/develop/src/main/proto/Ego.proto